service-tests/hhvm.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'hhvm', title: 'hhvm status' });
+module.exports = t;
+
+t.create('get symfony status (default branch - not tested)')
+    .get('/symfony/symfony.json')
+    .expectJSONTypes(Joi.object().keys({ name: 'hhvm', value: 'not tested' }));
```

Comment:
A bit of project convention: We use `.expectJSON()` if we're comparing the response to an object literal and `.expectJSONTypes()` if we're checking the type of the response keys or validating against a regex.